Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rust:1.59.0-alpine as builder
+RUN apk add libc-dev protoc
```

Comment:
I'm curious, what do we need a protobuf compiler for?

Reply:
console-subscriber pulls in prost, which requires protoc with its default features.